### PR TITLE
SDK-607: InApp Notification Rendering Controls

### PR DESCRIPTION
**Pull request by Sonal-Kachare**
- Added public APIs for suspending, discarding, and resuming InApp Notifications
- Updated iOS module file and added symlinks for Public header files in CleverTapSDK/include

### CODE REVIEW

File: CleverTapSDK/CleverTap+InApps.h [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@import Foundation;
+#import "CleverTap.h"
+
+@interface CleverTap (InApps)
```
Comment (Aditi3) [inline CleverTapSDK/CleverTap+InApps.h:4]: Rename `InApps` to `InAppNotification` to follow the existing naming convention.

Comment (Sonal-Kachare) [reply CleverTapSDK/CleverTap+InApps.h]: Updated

**Comment by Aditi3** [comment]
Please take a look at the conflicting files.

File: CleverTapSDK/CleverTap.m [hunk 2]
```diff
@@ -3749,6 +3764,36 @@ - (void)showInAppNotificationIfAny {
...
+    }
+}
+
+- (void)resumeInAppNotifications {
+    if ([[self class] runningInsideAppExtension]) {
+        CleverTapLogDebug(self.config.logLevel, @"%@: resumeInAppNotifications is a no-op in an app extension.", self);
+        return;
+    }
+    if (!self.config.analyticsOnly) {
+        self.inAppRenderingStatus = CleverTapInAppResume;
```
Comment (Aditi3) [inline CleverTapSDK/CleverTap.m:3783]: Please add logging(s) for our debugging purpose.

Comment (Aditi3) [reply CleverTapSDK/CleverTap.m:3783]: For example - "Resuming InApp Notifications"

File: CleverTapSDK/CleverTap.m [hunk 3]
```diff
@@ -3749,6 +3764,36 @@ - (void)showInAppNotificationIfAny {
     }
 }
 
+- (void)suspendInAppNotifications {
+    if ([[self class] runningInsideAppExtension]) {
+        CleverTapLogDebug(self.config.logLevel, @"%@: suspendInAppNotifications is a no-op in an app extension.", self);
+        return;
+    }
+    if (!self.config.analyticsOnly) {
+        self.inAppRenderingStatus = CleverTapInAppSuspend;
```
Comment (Aditi3) [inline CleverTapSDK/CleverTap.m:3773]: Add logging as `CleverTapLogDebug(self.config.logLevel, @"%@: Please Note - InApp Notifications will be suspended till resumeInAppNotifications() is not called again", self);`

File: CleverTapSDK/CleverTap.m [hunk 4]
```diff
@@ -1707,6 +1717,9 @@ - (BOOL)inNotificationQueue {
 - (void)_showNotificationIfAvailable {
     if ([[self class] runningInsideAppExtension]) return;
     
+    //If inAppRenderingStatus is suspended do not continue showing inApps
+    if (self.inAppRenderingStatus == CleverTapInAppSuspend) return;
```
Comment (Aditi3) [inline CleverTapSDK/CleverTap.m:1721]: Add logging as `InApp Notifications are set to be suspended, not showing the InApp Notification`

**Review by Aditi3 — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**